kbase-extension/static/kbase/js/widgets/appWidgets2/input/floatInput.js

```diff
@@ -89,11 +94,12 @@ define([
             return {
                 type: 'keyup',
                 handler: function (e) {
+                    const target = e.target;
```

Comment:
As in the int module, this fixes a test issue. Also, it's possible, though unlikely, that the target could disappear if the user is quick enough to trigger keyup and delete the cell / config tab before the event gets dispatched. This would be a silent error, but still an error.

Upcoming PR(s) will do this to the rest of the inputs that have a keyup handler.

Reply:
I wouldn't put anything past those dastardly narrative users!